addons/crm/models/crm_lead.py

```diff
@@ -164,6 +163,7 @@ class Lead(models.Model):
     lost_reason = fields.Many2one(
         'crm.lost.reason', string='Lost Reason',
         index=True, ondelete='restrict', tracking=True)
+    ribbon_message = fields.Char('Ribbon message', readonly=True)
```

Comment:
I would not store it. It is only for display purpose, it should not be stored in database.